content/kotlin/concepts/strings/terms/dropLast/dropLast.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'dropLast()'
+Description: 'Returns a string with n characters removed.'
+Subject:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The Kotlin method, **`.dropLast()`** returns a string with the last `(n)` characters removed.
+
+## Syntax
+
+```pseudo
+value string = "Example String"
+println(string.dropLast())
```

Comment:
This seems more like an example. You can just add a valid syntax, you can refer to the official documentation too but don't copy and paste it directly as it will be considered Plagiarism.